tests/plot/annotation/test.typ

```diff
@@ -44,3 +44,15 @@
     })
   })
 })
+
+#test-case({
+  import draw: *
+  set-style(rect: (stroke: none))
+
+  plot.plot(size: (6, 4), x-tick-step: 1, {
+    plot.add(domain: (100, 101), calc.sin)
```

Comment:
Can you change this to `(100, 100+2*calc.pi), x => calc.sin(x) + 50` and remove the annotation? Annotation "domains" for text are not correct, and I would like to not have their wrong behaviour in this test.